### PR TITLE
GoalTracker

### PR DESCRIPTION
**Pull request by davidfig**
I added a goal tracker to the word count. It places a green bar behind the word count to show progress. It is configurable through two settings (a word goal setting, and a color setting).

### CODE REVIEW

File: lib/wordcount-view.coffee [hunk 1]
```diff
@@ -24,4 +29,4 @@ class WordcountView extends View
   count: (text) ->
     words = text?.match(/\S+/g)?.length
     chars = text?.length
-    [words, chars]
+    [words, chars]
```
Comment (nesQuick) [inline lib/wordcount-view.coffee]: please add a newline here

**Comment by nesQuick** [comment]
Thank you for the contribution! I really appreciate it, that folks pick up the tasks and features they want to see in wordcount! That said, I'd love to merge this, but please remove this one newline. 

As a addition I'd like to propose a different look of the goaltracker. I this it should be a bit more subtle, like having a extra bat in the wordcount widget, a 90/10 split e.g. I attached a sketch to better transport my idea :pencil2: 

drawing

**Comment by davidfig** [comment]
I agree that subtly would be nice. Given the size of the panel (at least on my gutter), I worry, however, that by making it 10% of the height, it will not be visible. There's little room between the bottom of the letters and panel, and it would look weird overlapping the bottom part of the letter.

I'll play with it and see what I can do.

**Comment by nesQuick** [comment]
Awesome! Looking forward to what you will come up with! We could also make
the font 100% and the bar and userbg-color 90/10 but I assume that might be
a bit complex implementing.

**Comment by davidfig** [comment]
More work than I expected, but the height of the line is now configurable. I decided against the color gradient in your drawing since it's hard to know what percentage to assign to each of the colors, and making it user configurable likely would be confusing.

**Comment by nesQuick** [comment]
Yeah! I like it very much! Last though: I think making it configurable just adds more complexity then needed to the config, I like the 20, why not just setting it to 20? :)

**Comment by davidfig** [comment]
I agree on the added complexity, but I actually like it at 100%, which is why I decided to make it configurable so we'd both be happy :)

**Comment by nesQuick** [comment]
Alright, I see your point then. Let's see how many more options will pop up and when it become too cluttered this *might* change. Let;s get this in and bump a version. Thanks for helping out here!

**Comment by nesQuick** [comment]
just published 2.3.0

```
λ  atom-wordcount git:(master) apm publish minor
Preparing and tagging a new version ✓
Pushing v2.3.0 tag ✓
Publishing wordcount@v2.3.0 ✓
```

:tada:

**Pull request merged**